notebooks/information_extraction_raven.ipynb

```diff
@@ -0,0 +1,2726 @@
+{
```

Comment:
<p>You can say maybe "mock up function" instead of "fake function"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>